app/models/application_record.rb

```diff
@@ -1,3 +1,6 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+  EMAIL_REGEXP = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
+  ALPHA_REGEXP = /\A[a-z A-Z]+\z/
+  NUMERIC_REGEXP = /\A[0-9]+\z/
```

Comment:
If you want to validate if a field is a numeric one you can use its proper validation:

`:numericality => {:only_integer => true}`

Reply:
I want an string field, but which acept only number characters. I wont perform calculations with them.

Reply:
Please don't use " => ", this syntax is old

Reply:
why to have a string field that only accepts numbers?

Reply:
Can be a numeric code's instance, by example (PK).

Reply:
Agreed @matcheus, that was just the example, the correct implementation would be:

numericality: { only_integer: true }

Reply:
@vinibr3 it doesn't make sense in this context @vinibr3, unless you are validating a private key. Right now it seems that you are only using it on the `order` field which doesn't make sense to be an integer

Reply:
@vinibr3 Right now it seems that you are only using it on the order field which doesn't make sense to be an integer